docs/xxiprofileduringstartupphase.md

```diff
@@ -0,0 +1,49 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -XX:[+|-]IProfileDuringStartupPhase
+
+This option enables or disables the I-Profiler data collection of the profiling information during a VM startup.
+
+## Syntax
+
+        -XX:[+|-]IProfileDuringStartupPhase
+
+| Setting                          | Effect  |
+|----------------------------------|---------|
+|`-XX:+IProfileDuringStartupPhase` | Enable  |
+|`-XX:-IProfileDuringStartupPhase` | Disable |
+
+## Explanation
+
+The VM collects the I-Profiler information that is used to optimize methods at the time of JIT compilation. This interpreter profiling data is stored in the shared classes cache and improves the VM startup time. The VM uses an algorithm that includes the shared classes cache properties to decide whether to collect the information or not during a VM startup. You can overrule the algorithm result and control the collection of the profiling information during the startup phase by using the `-XX:[+|-]IProfileDuringStartupPhase` option.
```

Comment:
Again, I would replace I-Profiler with "interpreter profiler".
In the second sentence I don't it's clear why storing IProfiler information in SCC improves start-up time. Let me try something:

The VM collects interpreter profiling data that is used to optimize methods at the time of JIT compilation. To reduce the overhead of the interpreter profiler during the startup phase, the VM stores such profiling data in the Shared Class Cache (SCC), and uses heuristics regarding when to turn on data collection and when to rely on data collected by a previous VM and cached in the SCC. You can overrule the heuristics and control the collection of the profiling information during the startup phase by using the `-XX:[+|-]IProfileDuringStartupPhase` option.

- If the `-XX:+IProfileDuringStartupPhase` option is specified, the VM always collects interpreter profiling information during startup. Use this option to collect the latest profiling information for optimizing the methods and storing the information in the SCC. By using this option, the quality of the generated code may increase, leading do better long term throughput. On the downside, this option may increase the startup time of applications.
- If the `-XX:-IProfileDuringStartupPhase` option is specified, the VM does not collect interpreter profiling information during the startup phase. Use this option if the SCC already contains sufficient interpreter profiling information and therefore, you don't have to collect new information and affect startup time.

Reply:
> To reduce the overhead of the interpreter profiler during the startup phase,

Instead of "overhead", can we use CPU usage? Or could you suggest some other word? Thanks!

Reply:
Yes, "CPU usage" can be used instead.